### DISCUSSION TITLE
@ejabberd_systemd:init/1:94 Got no NOTIFY_SOCKET, notifications disabled

### DISCUSSION THREAD

**Discussion by Eirikr70** [processone/ejabberd#4219] [Questions] [answered]
Hello,
I have Ejabberd installed through ECS docker container. Each time I start the container, I get the following info
`2024-05-09 00:03:52.336866+00:00 [info] <0.516.0>@ejabberd_systemd:init/1:94 Got no NOTIFY_SOCKET, notifications disabled`
I haven't found any explanation about that message, explaining what kind of notification was disabled (so as to know if I want to implement them) nor any reference to NOTIFY_SOCKET in the docs.

**Comment by prefiks** [comment, marked as answer]
This is about socket that is used to communicate with systemd for it's watchdog feature. I don't think you can use it from docker.

**Reply by Eirikr70** [reply]
Thanks for that fast answer.
